commands/local_new.go

```diff
@@ -371,3 +377,19 @@ func forcePHPVersion(v, dir string) (string, error) {
 	os.Setenv("FORCED_PHP_VERSION", v)
 	return strings.Join(strings.Split(v, ".")[0:2], "."), nil
 }
+
+func createPhpVersionFile(v, dir string) error {
+	file := filepath.Join(dir, ".php-version")
+	f, err := os.OpenFile(file, os.O_WRONLY|os.O_CREATE|os.O_TRUNC, 0644)
+	if err != nil {
+		return errors.Wrapf(err, "unable to create %s", file)
+	}
+	if _, err = f.WriteString(v + "\n"); err != nil {
+		f.Close()
```

Comment:
If there was an error, the file has not been open, so it cannot be closed, right?

Reply:
That error was handled before right after `os.OpenFile`, at this point the file should be open, this is about handling write errors or I'm missing something? Maybe it's too verbose and I should use `defer`, but I've read not to (https://www.joeshaw.org/dont-defer-close-on-writable-files/). Maybe in this case it would be good enough though.